src/scss/components/_targid.scss

```diff
@@ -283,21 +297,30 @@ div.filmstrip {
       }
 
       > .parameters {
+        margin-bottom: 0;
+        grid-area: parameters;
         opacity: 1;
         transition: opacity 0.3s ease;
+
         p {
           margin: 5px 0;
         }
+
+        // add gap between wrapped elements
+        .form-inline {
+          display: inline-flex;
+          flex-wrap: wrap;
+          gap: 3px;
+        }
       }
 
       > .parameters:not(.hidden) ~ .inner {
+        position: relative;
         border-top: 1px solid $core_border;
         padding-top: 10px;
-        position: absolute;
-        top: 45px;
-        bottom: 0;
-        left: 0;
-        right: 0;
+        grid-area: inner;
+        height: 1px; // dummy height needed for the overflow-y
```

Comment:
I need some height for the overflow to work. Maybe you know a better solution?

Reply:
Might not be necessary. I changed it with #327.

Reply:
With 73545ea925f5d4a5d8bdf249205310c6a6a2e738 I switched the container to CSS grid. Afterwards it was possible to use [`height: 0; min-height: 100%;`](https://stackoverflow.com/a/58488563) to scale the TDP views to its parent container.